wandb/sdk/interface/artifacts.py

```diff
@@ -475,7 +475,9 @@ def add(self, obj: WBValue, name: str):
         ```
 
         Arguments:
-            obj: (wandb.WBValue) The object to add.
+            obj: (wandb.WBValue) The object to add. Currently support one of
```

Comment:
While you're adjusting these docstrings, could you shorten the brief description (L470-471) to a single line and put it directly underneath the `def`?

Our docs tool uses that first line as a "brief docstring" that gets handled specially, and we'll be doing this with all documented objects going forward.

Reply:
oh super cool - yeah 100% thanks for sharing

Reply:
Also, regarding using f-string, i think it can be achieved, but we will be moving to support all media types shortly.